index.html

```diff
@@ -45,6 +45,110 @@ <h3 id="tickets">Tickets</h3>
+            <tbody>
+              <tr>
+                <td>08:45 - 09:00</td>
+                <td>Get in</td>
+              </tr>
+              <tr>
+                <td>09:00 - 09:30</td>
+                <td><i>Breakfast</i></td>
+              </tr>
+              <tr>
+                <td>09:30 - 09:45</td>
+                <td><strong>Welcome to the CouchDB Day</strong></td>
+              </tr>
+              <tr>
+                <td>09:45 - 10:15</td>
+                <td><strong>Keynote by Jan Lehnardt</strong></td>
+              </tr>
+              <tr>
+                <td>10:15 - 10:30</td>
+                <td>Find your group you want to work in</td>
+              </tr>
+              <tr>
+                <td>10:30 - 12:30</td>
+                <td>Hands on - working in groups</td>
+              </tr>
+              <tr>
+                <td>12:30 - 13:30</td>
+                <td><i>Lunch</i></td>
+              </tr>
+              <tr>
+                <td>13:30 - 14:00</td>
+                <td><strong>CouchDB 2.0 - Klaus Trainer</strong></td>
+              </tr>
+              <tr>
+                <td>14:00 - 14:15</td>
+                <td><i>Break: Coffee and Cake</i></td>
+              </tr>
+              <tr>
+                <td>14:15 - 18:15</td>
+                <td>Hands on - working in groups</td>
+              </tr>
+              <tr>
+                <td>18:15 - 19:00</td>
+                <td><strong>How to implement token login for new users reusing the cookie system in CouchDB - Ólafur Arason</strong></td>
+              </tr>
+              <tr>
+                <td>19:00 - 19:15</td>
+                <td><strong>Robert and Andy say goodbye</strong></td>
+              </tr>
+              <tr>
+                <td>19:15 - ??:00</td>
+                <td>Party</td>
+              </tr>
+            </tbody>
+          </table>
+        </aside>
+
+        <aside class="right">
+          <h3>Proposed Groups</h3>
+          <table class="schedule">
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Description</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td>Fauxton</td>
+                <td>How to contribute to Fauxton?</td>
+              </tr>
+              <tr>
+                <td>CouchDB Core</td>
+                <td>How to contribute to the CouchDB core (Erlang)?</td>
+              </tr>
+              <tr>
+                <td>Use CouchDB</td>
+                <td>Hack an application with the usage of CouchDB ...</td>
+              </tr>
+              <tr>
+                <td>Third party CouchDB Tools</td>
+                <td>Build applications based on Hoodie, PouchDB and much more ...</td>
+              </tr>
+              <tr>
+                <td>The CouchDB Community</td>
+                <td>How to get involved in the various parts of the Apache CoucDB project?</td>
```

Comment:
How does managing an OpenSource Community work? How to get involved in the various parts of the Apache CouchDB project?